api/v1/openapi.yaml

```diff
@@ -1058,6 +1058,9 @@ definitions:
       unknown:
         description: Number of unknown samples.
         type: integer
+      numlisteners:
```

Comment:
How about renaming into `nlisteners` for consistency with `npages` above, or just `listeners` for consistency with `cpus`, `lost`, and `unknown` above?

Reply:
`nlisteners` sounds good.